plugins/Installation/templates/finished.twig

```diff
@@ -20,10 +20,10 @@
     {% if areAdsForProfessionalServicesEnabled %}
         <h3>{{ 'Installation_ProfessionalServicesAdTitle'|translate }}</h3>
         <p>
-            {{ 'Installation_ProfessionalServicesfessionalServicesAdText'|translate }}
+            {{ 'Installation_ProfessionalServicesfessionalServicesAdText'|translate('<a rel="noreferrer noopener" target="_blank" href="https://plugins.matomo.org/premium?pk_campaign=App_ProfessionalServices&pk_medium=App_InstallationFinished&pk_source=Matomo_App">','</a>','<a rel="noreferrer noopener" target="_blank" href="https://matomo.org/support-plans/?pk_campaign=App_ProfessionalServices&pk_medium=App_InstallationFinished&pk_source=Matomo_App">')|raw }}
         </p>
         <p>
-            {{ 'Installation_ProfessionalServicesfessionalDiscoverHow'|translate('<a rel="noreferrer noopener" target="_blank" href="' ~ linkToProfessionalServices ~ '">','</a>')|raw }}
+            {{ 'Installation_ProfessionalServicesfessionalDiscoverHow'|translate('<a rel="noreferrer noopener" target="_blank" href="https://matomo.org/matomo-cloud/?pk_campaign=App_ProfessionalServices&pk_medium=App_InstallationFinished&pk_source=Matomo_App">','</a>')|raw }}
```

Comment:
When you don't use the template variable `linkToProfessionalServices` anymore, it can be remove in the controller as well. On the other side you could also have variables for all URLs and add methods to the Advertising class to generate them like
https://github.com/matomo-org/matomo/blob/8a2b6425bbaa24558a692b7af02375d108a23b25/core/ProfessionalServices/Advertising.php#L56-L67